Assertions updates

jmcmichael opened the pull request:
Minor updates to the recent assertions features deploy.

susannasiebert commented on src/app/views/events/common/evidenceGridIdCell.tpl.html:
```diff
@@ -3,7 +3,7 @@
      tooltip-placement="right"
      tooltip-append-to-body="true">
-  <span class="btn-block" ng-class="{ 'statusAccepted': row.entity['status'] === 'accepted', 'statusSubmitted': row.entity['status'] === 'submitted', 'statusRejected': row.entity['status'] === 'rejected' }">
+  <span class="btn-block" ng-class="{ 'statusAccepted': row.entity['status'] === 'accepted', 'statusSubmitted': row.entity['status'] === 'submitted', 'statusRejected': row.entity['status'] === 'rejected' || row.entity['status'] === 'assertion rejected'  }">
```
This is not entirely necessary because we updated the assertion status values before the first deploy so no assertion in the live database would have a status of 'assertion rejected'.

susannasiebert commented:
We also need to add this warning when editing an assertion.

susannasiebert commented:
Since we dropped the assertion status column in the browse assertion grid, we don't have a way to filter on that anymore. Maybe we should add the little drop down in the top right corner to filter on status, that we have in other places.

jmcmichael commented:
- removed the unnecessary status check from the status cell template
- added warning to edit assertion form

Adding the grid menu for toggling the display of non-accepted records is a good idea.  However, the browse grid re-generates its column options when the entity type changes, and the grid menu config isn't currently incorporated into the code that does the re-generating. It's doable, but would take a little extra time to accomplish.

susannasiebert commented:
Yeah, I don't think this is a blocker for this PR.
